src/main/java/programminglife/gui/controller/GuiCreateBookmarkController.java

```diff
@@ -44,16 +47,20 @@ private void initButtons() {
             try {
                 inputCenter = Integer.parseInt(txtId.getText());
                 inputRadius = Integer.parseInt(txtRadius.getText());
+                bookmarkName = txtBookmarkName.getText();
+                System.out.println(bookmarkName);
+                bookmarkDescription = txtDescription.getText();
+                Bookmark bookmark = new Bookmark(name, guiController.getFile().getAbsolutePath(),
+                        inputCenter, inputRadius, bookmarkName, bookmarkDescription);
                 if (!guiController.getGraphController().getGraph().contains(inputCenter)) {
                     Alerts.warning("Center node is not present in graph");
                 } else if (inputRadius < 0) {
                     Alerts.warning("Radius can only contain positive integers");
-                } else if (txtBookmarkName.getText().length() == 0) {
+                } else if (bookmarkName.length() == 0) {
                     Alerts.warning("Bookmark must contain a name");
-                } else if (!txtBookmarkName.getText().matches("[\\S.*]?\\S")) {
+                } else if (!bookmarkName.matches("\\S|\\S.*\\S")) {
```

Comment:
Actually, I think Toine solution was better, except that he should have used `(...)` instead of `[...]`.